app/models/view_model/auctions_index.rb

```diff
@@ -0,0 +1,33 @@
+module ViewModel
+  class AuctionsIndex < Struct.new(:current_user, :auctions_query)
+    def auctions
+      @auctions ||= auctions_query.map  {|auction| Presenter::Auction.new(auction) }
+    end
+
+    def active_auction_count
+      auctions.count { |i| i.start_datetime < Time.now && Time.now < i.end_datetime }
+    end
+
+    def upcoming_auction_count
+      auctions.count { |i| Time.now < i.start_datetime }
+    end
+
+    def header_partial
+      if current_user && current_user.sam_account?
+        '/components/sam_verified_header'
```

Comment:
Should these be in `/components/` like I had placed some of them earlier or is there a better place for them?